interface/chat/components/containers/chatbot-container.jsx

```diff
@@ -0,0 +1,76 @@
+import ChatOutput from "../chatbot/chat-output";
+import ChatInput from "../chatbot/chat-input";
+import SendChatInput from "../menu/chatbotButtons/send-chat-input";
+
+import ClearButton from "../menu/accountButtons/clear-button";
+
+import React, {useState} from "react";
+
+export default function Chatbot() {
+    // these have to be here or else it breaks :)
```

Comment:
could you be more specific? These variables don't appear to be used, but I could be mistaken. If they are globals of some sort, should they be declared here?